pkg/util/workload/deployment.go

```diff
@@ -0,0 +1,131 @@
+package workload
```

Comment:
I think we copy and modified some functions from k/k. So we need a note here.

It is recommended to put the copied codes to one separated file with their file header(copyright).

Reply:
OK, I just lifted the code and made some revision. The origin code use API instead of lister and informer.